third_party/nvfuser/csrc/ops/alias.cpp

```diff
@@ -183,7 +185,7 @@ TensorView* squeeze(TensorView* x, const std::vector<int64_t>& sizes) {
   const auto ndims = static_cast<int>(x->domain()->noReductions().size());
 
   TORCH_INTERNAL_ASSERT(
-      ndims == sizes.size(),
+      ndims == int(sizes.size()),
```

Comment:
Does this need to be `int64_t` to be consistent?

Reply:
ndims is `int`, so trying to keep things the same. downcast should also be safe since this is only the rank.